### PR TITLE
Implemented get_all_versions_python_package method for Dgraph

### PR DESCRIPTION
**Pull request by pacospace**

### CODE REVIEW

File: thoth/storages/graph/dgraph.py [hunk 1]
```diff
@@ -346,7 +346,37 @@ def get_all_versions_python_package(
...
+            {
+                f(func: has(%s)) @filter(eq(package_name, %s)%s) {
+                    package_version
+                    index_url
+                }
+            } 
+            """ % (PythonPackageVersion.get_label(), package_name, q)
+
+        result = self._query_raw(query)
+        return [[python_package['package_version'],python_package['index_url']] for python_package in result["f"][0]]
```
Comment (fridex) [inline thoth/storages/graph/dgraph.py]: The signature expects tuple so explicit cast needed, also there is no need to explicitly obtain package_version and index_url as only those two are part of result.

```python
        return [tuple(python_package) for python_package in result["f"][0]]
```

File: thoth/storages/graph/dgraph.py [hunk 2]
```diff
@@ -346,7 +346,37 @@ def get_all_versions_python_package(
         without_error: bool = False,
     ) -> List[Tuple[str, str]]:
         """Get all versions available for a Python package."""
-        return []
+        package_name = self.normalize_python_package_name(package_name)
```
Comment (fridex) [inline thoth/storages/graph/dgraph.py]: :+1:

File: thoth/storages/graph/dgraph.py (hunk 1 above)
Comment (pacospace) [reply thoth/storages/graph/dgraph.py]: yes but python_package is still a dict ``` return [tuple(python_package.values() for python_package in result["f"]] ````

Comment (fridex) [reply thoth/storages/graph/dgraph.py]: Agh, I meant:

```python
        return [tuple(python_package['package_version'], python_package['index_url']) for python_package in result["f"][0]]
```

**Pull request merged**